Scripts/script-ChangeRemediationSLAOnSevChange.yml

```diff
@@ -0,0 +1,60 @@
+fromversion: 4.1.0
+commonfields:
+  id: changeremediationslaonsevchange
+  version: -1
+name: ChangeRemediationSLAOnSevChange
+script: |
+  import datetime
+
+  # ##### Help #####
+  # This is an example script. The script is used to change the Remediation SLA of an incident, when the severity of the incident changes for any reason. Please copy this script and make changes to your liking.
+  # The Configuration section is there to help you easily configure the script with your desired SLAs.
+
+  # The CRITICAL_SLA field defines the number of minutes that you would want an incident with critical severity to have, in its Remediation SLA field.
+  # The NONCRITICAL_SLA field defines the number of days that you would want an incident with non-critical severity to have, in its Remediation SLA field.
+  # The NONCRITICAL_SLA field can also be configured in minutes if you want.
+
+  # Note that the SLA can be set with a number that represents minutes instead of days, like so: demisto.executeCommand("setIncident",{'sla': 30, "slaField":"remediationsla"})
+  # but it can also be set with a number that represents a complete date and time structure, like so: demisto.executeCommand("setIncident",{'sla': 2018-12-26T12:10:24Z, "slaField":"remediationsla"})
```

Comment:
Are you sure it's the right date format? just tell them to look at the place in the code that does that and copy it.